front/src/styles/scss/applications/stdcmV2/_card.scss

```
@@ -61,39 +61,101 @@
   }
 
   .stdcm-v2-card__body {
-    padding: 0.688rem 1rem 0.8125rem 1.5rem;
-    // TODO: remove this style after ui-core component is implemented
-    .suggestions * label,
-    .stdcm-v2-via-stop-for * label {
+    padding: 14px 24px 16px 8px;
```

```suggestion
    padding: 14px 24px 16px 24px;
```
You can adjust the left padding here instead of adding it on each element of the card :)
And maybe set the sizes in rem too ?